org.eclipse.scanning.points/src/org/eclipse/scanning/points/RandomOffsetGridGenerator.java

```diff
@@ -27,11 +28,20 @@
 	protected void validateModel() {
 		super.validateModel();
 		if (!(model instanceof RandomOffsetGridModel)) {
-			throw new ModelValidationException("The model must be a RandomOffsetGridModel", model, "offset"); // TODO Not really an offset problem.
+			throw new ModelValidationException("The model must be a " + RandomOffsetGridModel.class.getSimpleName(),
+					model, "offset"); // TODO Not really an offset problem.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)